src/order/symm_diff.lean

```diff
@@ -55,6 +55,8 @@ rfl
 
 lemma symm_diff_eq_xor (p q : Prop) : p ∆ q = xor p q := rfl
 
+lemma symm_diff_eq_bxor (p q : bool) : p ∆ q = bxor p q := by cases p; cases q; refl
```

Comment:
Is there a way of deducing this from the previous (or vice versa) by coercing the bools to Props then using something about xor = bxor?